### PR TITLE
Comments on things we say about VISION, et al

### PR DESCRIPTION
**Pull request by gonuke**
These are two comments regarding negative claims we make about other tools, particularly VISION.  I raise them to make sure we are all on the same page about what they mean and how we would back up these statements.

### CODE REVIEW

File: fundamentals/conclusion.tex [hunk 1]
```diff
@@ -16,6 +16,12 @@ \section{Conclusions}
 %Open paradigm allows community oversight, testing, verification
 %Enables apples to apples comparisons
 
+% gonuke: I am sensitive to making the implication that VISION (or others) has
```
Comment (katyhuff) [inline fundamentals/conclusion.tex]: I've actually recently taken uncertain provenance out. I'm not sure what that was about.

Comment (gonuke) [inline fundamentals/conclusion.tex]: +1

File: fundamentals/fcsreqs.tex [hunk 2]
```diff
@@ -26,6 +26,12 @@ \subsubsection{Discrete Facilities and Materials}
 
 \subsubsection{Open Access}
 
+% gonuke: I am perhaps oversensitive, but I perceive the status of 
+%         VISION to be constantly shifting.  I am thus unsure whether
+%         ``license agreemnts or instutional approval'' are currently
+%         required for access to VISION.  I raise it here to satisfy
+%         my desire to have it categorically confirmed by someone.
```
Comment (katyhuff) [inline fundamentals/fcsreqs.tex]: This is a good point. I think at this point, you do still have to ask them for it, but you may not have to do too much paperwork. Ultimately though, it relies on powersim, which I think does require a significant license, doesn't it?

Comment (gonuke) [inline fundamentals/fcsreqs.tex]: True, but I think we make that point separately in the next sentence for CAFCA.

Comment (katyhuff) [inline fundamentals/fcsreqs.tex]: This may be worth getting an update from INL about what is necessary for using VISION.

**Comment by katyhuff** [comment]
I'm going to turn this vision comment into an issue.

File: fundamentals/fcsreqs.tex (hunk 2 above)
Comment (gidden) [inline fundamentals/fcsreqs.tex]: The powersim license was ~$800 if I recall correctly

On Sun, Feb 22, 2015 at 1:31 PM, Katy Huff <notifications@github.com> wrote:

> In fundamentals/fcsreqs.tex
> <https://github.com/cyclus/fundamentals-paper/pull/82#discussion_r25135501>
> :
>
> > @@ -26,6 +26,12 @@ \subsubsection{Discrete Facilities and Materials}
> >
> >  \subsubsection{Open Access}
> >
> > +% gonuke: I am perhaps oversensitive, but I perceive the status of
> > +%         VISION to be constantly shifting.  I am thus unsure whether
> > +%         ``license agreemnts or instutional approval'' are currently
> > +%         required for access to VISION.  I raise it here to satisfy
> > +%         my desire to have it categorically confirmed by someone.
>
> This is a good point. I think at this point, you do still have to ask them
> for it, but you may not have to do too much paperwork. Ultimately though,
> it relies on powersim, which I think does require a significant license,
> doesn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/fundamentals-paper/pull/82/files#r25135501>.
>



-- 
Matthew Gidden
Ph.D. Candidate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

Comment (rwcarlsen) [inline fundamentals/fcsreqs.tex]: $800 for an academic single-use license.  As far as I understand (which may
not be 100% correct), if you want to deploy in parallel on many machines,
then you either need multiple licenses or must buy their full developer sdk
(which is more expensive) that comes with a license server and other such
fun.

**Pull request closed without merging**